chainer_pytorch_migration/ignite/extensions.py

```diff
@@ -296,9 +298,10 @@ def serialize(self, serializer):
 
         # Lets save torch objects using torch interface
         if isinstance(serializer, chainer.serializer.Serializer):
-            snap_path = self._chainer_trainer.get_extension(
-                '_Snapshot').filename
-            snap_path = snap_path.format(self)
+            name, ext = self.cur_ext
+            if '_MultiNodeSnapshot' in name:
```

Comment:
How about checking the name of the class?
```suggestion
            if type(ext).__name__ == '_MultiNodeSnapshot':
```

Reply:
Good catch